src/app/api/dashboard/feedback/route.js

```diff
@@ -114,27 +113,30 @@ export const GET = async (req) => {
         )
       );
     }
-    snapshot.forEach((doc) => {
-      const {
-        rating,
-        additionalComments,
-        eventSource,
-        improvements,
-        notBeneficial,
-        helpful,
-        status,
-      } = doc.data();
-      output.push({
-        uid: doc.id,
-        rating,
-        additionalComments,
-        eventSource,
-        improvements,
-        notBeneficial,
-        helpful,
-        status,
-      });
-    });
+
+    await Promise.all(
```

Comment:
promise.all on a map won't have any performance benefits

Reply:
There isn't much of an optimization we can make on the GET call since its already optimized via the query